lambda/rotator-snowflake/rotate/rotate.go

```diff
@@ -0,0 +1,169 @@
+package rotate
+
+import (
+	"bytes"
+	"context"
+	"crypto/rsa"
+	"crypto/x509"
+	"database/sql"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/chanzuckerberg/go-misc/keypair"
+	"github.com/chanzuckerberg/go-misc/lambda/rotator-snowflake/setup"
+	"github.com/chanzuckerberg/go-misc/snowflake"
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+	"github.com/xinsnake/databricks-sdk-golang/aws"
+	"github.com/xinsnake/databricks-sdk-golang/aws/models"
+)
+
+func getUsers() ([]string, error) {
+	usersList := os.Getenv("CURRENT_USERS") //Comma-delimited users list
+	userSlice := strings.Split(usersList, ",")
+	return userSlice, nil
+}
+
+func buildSnowflakeSecrets(connection *sql.DB, username string, privateKey *bytes.Buffer) (map[string]string, error) {
+	userQuery := fmt.Sprintf(`SHOW USERS LIKE '%s'`, username)
+
+	connectionRow := snowflake.QueryRow(connection, userQuery)
+
+	snowflakeUser, err := snowflake.ScanUser(connectionRow)
```

Comment:
```suggestion
	userQuery := fmt.Sprintf(`SHOW USERS LIKE '%s'`, username)
	connectionRow := snowflake.QueryRow(connection, userQuery)
	snowflakeUser, err := snowflake.ScanUser(connectionRow)
```
I try to use whitespace to group/separate logically similar pieces of code. These all seem pretty related so might be worth removing the whitespace to hint readers these are all closely related

Reply:
Oh my gosh, the linter told me to do this :|

Reply:
huh :confused:

Reply:
I agree with you though! I'll do this